zerver/openapi/zulip.yaml

```diff
@@ -4207,6 +4207,21 @@ paths:
                                       description: |
                                         The day of the week when the organization will send
                                         its weekly digest email to inactive users.
+                                    direct_message_initiator_group:
+                                      type: integer
+                                      description: |
+                                        The ID of the [user group](/api/get-user-groups) whose members are
+                                        allowed to initiate direct message thread.
+
+                                        **Changes**: New in Zulip 8.0 (feature level 238).
+                                    direct_message_permission_group:
+                                      type: integer
+                                      description: |
+                                        The ID of the [user group](/api/get-user-groups) of which at least one
+                                        member must be included as sender or recipient in all personal and
+                                        group direct messages.
+
+                                        **Changes**: New in Zulip 8.0 (feature level 240).
```

Comment:
These **Changes** entries should probably have an entry like:

> Previously, access to send direct messages was controlled by the `private_message_policy` realm setting, which supported values of 1 (enabled) and 2 (disabled).

Reply:
I have added them now.